pkg/kev/config/schema.go

```diff
@@ -58,8 +59,8 @@ var ServicesSchema = map[string]interface{}{
 		LabelServiceExposeTLSSecret:                      map[string]interface{}{"type": "string"},
 	},
 	"required": []string{
-		LabelWorkloadLivenessProbeCommand,
 		LabelWorkloadReplicas,
+		LabelWorkloadLivenessProbeType,
```

Comment:
For added visibility perhaps we should still show the basic settings for the chosen liveness probe type? So if type was `command` then we also show the command. If type was `http` then we would probably need to show the basic parameters for it like `path` and `port`. Thoughts @ezodude?

Reply:
Agreed (that's why I wanted to see an example of an environment compose file).

Reply:
I don't think we can add them here because of how limited the json validation is but I think it's worth another ticket to investigate and either how to do it in json validation or replace the validation with something more capable. I think this might be possible with the current solution but will need a bit more time to figure that one out

Reply:
Alternative to all that would be to hide everything but the replicas number by default. This means that internally probe type will default to either `command` (as we already have a default for it if not specified in docker compose) or `disabled` (less ideal as we should encourage users to specify those things).

Reply:
TBH, I think most docker-compose users DO NOT utilise the `healthcheck` attributes. [See this search](https://github.com/search?q=healthcheck+extension%3Ayaml+filename%3Adocker-compose.yaml&type=Code&ref=advsearch&l=&l=), only 2K compose files use `healthcheck` on Github.

So minimising config to just `replicas` makes sense (as we'll almost always default to a dummy healthcheck).

Reply:
yeh, as a context / background information this is the current flow:

If [docker-compose healthcheck](https://docs.docker.com/compose/compose-file/compose-file-v3/#healthcheck) is defined on any given application stack service then it'll be detected / extracted and translated to equivalent K8s liveness probe.

If the `healthcheck` (as described above) is not present on a service, then we assume a default - currently it's an exec command based liveness probe with a default value defined [here](https://github.com/appvia/kev/blob/master/pkg/kev/config/defaults.go#L125).

With proposed changes we simply apply the same inference mechanism i.e. if the `healtcheck` is defined in docker compose then we'll translate that to equivalent `exec / command` based liveness probe behind the scenes. 

Equally, if the `healthcheck` wasn't detected then we could use default (`command` type) liveness probe command value as this would be the closest link to docker-compose specification - currently only command based healthchecks are supported... OR, if this makes more sense, `disable` liveness probe entirely (albeit less ideal).

Whatever is inferred behind the scenes can be changed by the user in an environment specific docker compose override file and these settings obviously get persisted there. 

Example of how environment specific configuration could look like (minimum set of parameters visible)
```yaml
version: "3.7"
services:
  postfacto:
    labels:
      kev.workload.replicas: "1"
```
The above doesn't change the way default liveness probe gets constructed behind the scenes (based on what was detected in docker-compose and kev defaults) but used can then adjust the defaults with custom settings e.g.

```yaml
version: "3.7"
services:
  postfacto:
    labels:
      kev.workload.replicas: "1"
      kev.workload.liveness-probe.type: "http"
      kev.workload.liveness-probe.http.path: "/"
      kev.workload.liveness-probe.http.port: "3000"
```

Reply:
I will create a new ticket to track "exec" instead of "command" and another one to track "tcp" implementation. Also one for dependent base labels  ie exec means command is added, thtp means port and path are added

Reply:
https://github.com/orgs/appvia/projects/5#card-56766770
https://github.com/orgs/appvia/projects/5#card-56766720
https://github.com/orgs/appvia/projects/5#card-56766671